Datahub.Core/Services/IProjectDatabaseService.cs

```diff
@@ -0,0 +1,11 @@
+using System.Threading.Tasks;
+using Microsoft.Azure.Services.AppAuthentication;
+
+namespace NRCan.Datahub.Shared.Services
+{
+    public interface IProjectDatabaseService
```

Comment:
is there any technical reason we need this interface?

Reply:
It's good practice in general, and in this case we can have a dummy version for offline mode